### DISCUSSION TITLE
cpu可以正常训练，但是gpu训练的时候有时候loss异常，有时候训练突然被中断

### DISCUSSION THREAD

**Discussion by 2013358072** [PaddlePaddle/PaddleOCR#13547] [Q&A]
### Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.

- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.

- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### Bug

loss异常的代码在./losses/det_basic_loss.py文件的136行报错，检查过数据集没有问题，参数也没有问题，换了好几个模型都是同样的问题
训练突然中断的情况没有报错

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by GreatV** [comment]
换个显卡试试

**Reply by 2013358072** [reply]
我换了一个8张t4的机器，还是同样的问题

**Comment by 2013358072** [comment]
谢谢，我试一下
